projects/fsastorefrontstyles/scss/base/comparison-table.scss

```diff
@@ -1,135 +1,135 @@
-cx-fs-comparison-table-container {
+%cx-fs-comparison-table-container {
```

Comment:
Adding % next to custom tag names `cx-fs-comparison-table-container, cx-fs-comparison-table-tab,
  cx-fs-comparison-table-panel, cx-fs-comparison-table-panel-item` was the whole change